samples/snippets/pandas_date_and_time.py

```diff
@@ -0,0 +1,84 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+def pandas_date_and_time():
+    # fmt: off
```

Comment:
Is black formatting too invasive here? Re-formatting multiline `pd.Series(...)` snippets is IMO not that bad, or is there something else?

(might be subjective, don't put too much weight on it)

Reply:
These examples show up in HTML pages where there isn't much width.  Blacken makes examples so wide that readers have to scroll horizontally, which is mean. :)

Reply:
Ah, OK, that's indeed terrible much. :)